### PR TITLE
Update Quickstart section in README

### PR DESCRIPTION
**Pull request by arvindkrishnakumar-okta**
### Issue

https://github.com/okta/okta-spring-boot/issues/186

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,16 +6,31 @@
...
 
+## Quickstart
+
+* Create a Springboot Application with [Spring initializr](https://start.spring.io/):
+```bash
+curl https://start.spring.io/starter.zip -d dependencies=web,okta -d javaVersion=11 -o <<yourProjectName>>.zip
+```
+* Configure it with [Okta CLI](https://github.com/oktadeveloper/okta-cli/blob/master/README.md):
+```bash
+unzip <<yourProjectName>>.zip && cd <<yourProjectName>> && okta apps create
```
Comment (bdemers) [inline README.md:27]:
```suggestion

curl https://start.spring.io/starter.tgz \
  -d dependencies=web,okta \
  -d baseDir=<yourProjectName> | tar -xzvf -
  
cd <yourProjectName>
```

* Configure it with [Okta CLI](https://github.com/oktadeveloper/okta-cli/blob/master/README.md):
```bash
okta apps create
```

Comment (bdemers) [reply README.md:27]: Java 11 is now the default on start.spring.io

**Review by bdemers — APPROVED** [review]
minor change suggested

File: README.md (hunk 1 above)
Comment (arvindkrishnakumar-okta) [reply README.md]: looks like zip is the only archive format supported (can't find tgz in spring doc at https://docs.spring.io/initializr/docs/0.6.0.RELEASE/reference/html/command-line.html).

Comment (arvindkrishnakumar-okta) [reply README.md]: addressed.

**Pull request merged**